src/experiments.js

```diff
@@ -36,8 +36,16 @@ const COOKIE_EXPIRATION_INTERVAL = COOKIE_MAX_AGE_DAYS * 24 * 60 * 60 * 1000;
 /** @const {string} */
 const CANARY_EXPERIMENT_ID = 'dev-channel';
 
-/** @const {!Object<string, boolean>} */
-const EXPERIMENT_TOGGLES = Object.create(null);
+/** @type {Object<string, boolean>} */
+let toggles;
```

Comment:
I am a tiny bit worried about this and similar ones ( `templateTagSupported` ) being accessed directly. We should hide them more. Either by scary names ( `uninitializedFoo` , `fooCache` ) or hide them all inside another object